aiidalab_widgets_base/structures.py

```diff
@@ -943,6 +1018,75 @@ def _to_standard_cell(
             pbc=[True, True, True],
         )
 
+    @tl.observe("structure")
+    def _observe_structure(self, change):
+        """Update cell after the structure has been modified."""
+        if change["new"] is not None:
+            cell_parameters = change["new"].cell.cellpar()
+            for i in range(6):
+                self.cell_parameters.children[i].children[1].value = round(
+                    cell_parameters[i], 4
+                )
+        else:
+            for i in range(6):
+                self.cell_parameters.children[i].children[1].value = 0
+        # reset transformation matrix
+        self.reset_cell_transformation_matrix()
```

Comment:
```suggestion
```
I suggest not resetting the matrix automatically. The user will lose the information they entered.

I think it is safer to do that only when pressing the button.

Reply:
I changed as you suggested.